packages/twenty-front/src/modules/object-record/record-table/record-table-section/components/RecordTableRecordGroupSection.tsx

```diff
@@ -0,0 +1,115 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { motion } from 'framer-motion';
+import { useCallback } from 'react';
+import { IconChevronUp, isDefined, Tag } from 'twenty-ui';
+
+import { useCurrentRecordGroupId } from '@/object-record/record-group/hooks/useCurrentRecordGroupId';
+import { recordGroupDefinitionFamilyState } from '@/object-record/record-group/states/recordGroupDefinitionFamilyState';
+import { RecordGroupDefinitionType } from '@/object-record/record-group/types/RecordGroupDefinition';
+import { isRecordGroupTableSectionToggledComponentState } from '@/object-record/record-table/record-table-section/states/isRecordGroupTableSectionToggledComponentState';
+import { numberOfTableRowIdsByGroupComponentFamilySelector } from '@/object-record/record-table/states/selectors/numberOfTableRowIdsByGroupComponentFamilySelector';
+import { numberOfvisibleTableColumnsComponentSelector } from '@/object-record/record-table/states/selectors/numberOfVisibleTableColumnComponentSelector';
+import { useRecoilComponentFamilyStateV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentFamilyStateV2';
+import { useRecoilComponentFamilyValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentFamilyValueV2';
+import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
+import { useRecoilValue } from 'recoil';
+
+const StyledTrContainer = styled.tr`
+  cursor: pointer;
+`;
+
+const StyledChevronContainer = styled.td`
+  border-bottom: 1px solid ${({ theme }) => theme.border.color.light};
+  color: ${({ theme }) => theme.font.color.secondary};
+  text-align: center;
+  vertical-align: middle;
+`;
+
+const StyledTotalRow = styled.span`
+  color: ${({ theme }) => theme.font.color.tertiary};
+  margin-left: ${({ theme }) => theme.spacing(2)};
+  text-align: center;
+  vertical-align: middle;
+`;
+
+const StyledRecordGroupSection = styled.td`
+  border-bottom: 1px solid ${({ theme }) => theme.border.color.light};
+  padding-bottom: 6px;
```

Comment:
use theme here (I'm a bit surprised by the 6px value, is this in Figma?
We usually have a 4px scale for paddings

Reply:
Yes it's what I have on Figma, strange

Reply:
@Bonapara FYI